### ISSUE TITLE
Enquiry: Integration with Jenkins

### ISSUE THREAD

**Issue by sarathsp23** [testvagrant/optimusTemplate#56]
Hi , 
Can I integrate the project with Jenkins and have the reports generated and published for Jenkins or any CI/CD tool?

**Comment by KrishnB** [comment]
Yes you can, please use the gradle command in the yml file or job command `./gradlew runDistribution - DtestFeed=<yourtestfeedname>`

**Comment by sarathsp23** [comment]
The issue have been noticing is that (this is also for the local instance as well), if the Mongo db is not up and running , it is not able to start the mongo and the framework just gets stuck indefinitely.
